docs/API_DESIGN.md

```diff
@@ -624,27 +623,24 @@ CLASS DidJwkCreateOptions
 
 > [!NOTE]
 >
-> The `CONSTRUCTOR(domain: string, public_jwk: Jwk)` does not publish the DID Document to a host, but merely creates the instance of the `did:web` in the local scope. 
+> The `create` does not publish the DID Document to a host, but creates the instance of the `did:web` `BearerDid` in the local scope. 
 
 ```pseudocode!
 CLASS DidWeb
-  CONSTRUCTOR(domain: string, public_jwk: Jwk)
-  CONSTRUCTOR(uri: string)
+  STATIC METHOD create(domain: string, options: DidWebCreateOptions?): BearerDid
   STATIC METHOD resolve(uri: string): ResolutionResult
```

Comment:
outside the scope of this PR, but curious if you've given thought to whether did web resolve, did dht resolve, and did dht publish should be async?

Reply:
yeah I think that's right actually, we should do that ... async programming in the binded world maybe needs a bit more thought 🤔 

https://github.com/TBD54566975/web5-rs/issues/309